climesync.py

```diff
@@ -71,12 +73,11 @@
         print getattr(ts, "__init__")(baseurl)
 
     if choice == "s":
-        if not username:
-            username = raw_input("username: ")
-        if not password:
-            password = raw_input("password (plaintext): ")
-        auth_type = "password"
-        pp.pprint(ts.authenticate(username, password, auth_type))
+        username = arg_username if arg_username else raw_input("username: ")
+
+        password = arg_password if arg_password else raw_input("password: ")
```

Comment:
What happens if you sign in like so:
```
./climesync.py -c <baseurl> -u user1 -p pass1
```
but then you want to change your user (think admin switching to his own account)
```
>> s
hey! you are already signed in because arg_username and arg_password are set.  
Here is a brand new token for the same user you set in the command line args!
```